solidity/dashboard/src/services/token-grants.service.js

```diff
@@ -218,10 +219,28 @@ const getOperatorsFromManagedGrants = async (web3Context) => {
   return operators
 }
 
+const fetchGrantById = async (web3Context, grantId) => {
+  const id = new BigNumber(grantId)
```

Comment:
Can we validate `grantId` in the if statement? Do we actually need to convert it to BigNumber since later we convert it to string?
`return await getGrantDetails(id.toString(), web3Context, true)`

Reply:
With `BigNumber` because we can easily check if the provided grntId is an integer or negative value. In that case, a `grantId` could be of type `string/number/BigNumber.

Reply:
If a `grantId` will be `asd`, `BigNumber.isInteger()` returns `false`.